example_configs/radicale.md

```diff
@@ -0,0 +1,27 @@
+#Configuration of RADICALE authentification with lldap.
+
+# See how to implement the correct plugin (v0.3) for radicale : https://github.com/shroomify-it/radicale-auth-ldap-plugin
+
+# Full docker-compose stack : https://github.com/shroomify-it/docker-deploy_radicale-agendav-lldap
+
+[auth]
+type = radicale_auth_ldap
+ldap_url = ldap://lldap:3890  
+ldap_base = dc=example,dc=domain,dc=com
```

Comment:
For configs we usually use dc=example,dc=com
It's a bit shorter, and mainly more consistent.